app/views/home/about.html.erb

```diff
@@ -74,7 +74,7 @@
   </p>
 
   <p>
-  Some things that are off-topic here but popular on larger, similar sites: entrepreneurship, management, news about companies that employ a lot of programmers, investing, world events, anthropology, and defining the single morally correct economic and political system for the entire world when we can't even settle tabs vs. spaces.
+  Some things that are off-topic here, but popular on larger, similar sites: entrepreneurship, management, news about companies that employ a lot of programmers, investing, world events, anthropology, and defining the single morally correct economic and political system for the entire world when we can't even settle tabs vs. spaces.
```

Comment:
I can see why one might set off this clause, but I prefer not to break up this sentence.

Reply:
It's undone.